### ISSUE TITLE
Issue Design: Explain principle of Open Humans in wireframe

### ISSUE THREAD

**Issue by katoss** [alan-turing-institute/AutSPACEs#321]
## Summary
The external login via open humans and the whole concept of authorisation of data usage for projects might be confusing without further explanation. The wireframe should include information and instructions regarding this process.

## Suggestions for tasks that can fix this issue

- short and easy to understand information text about the concept of open humans and why the login is performed via an external platform
- short instructions on how the registration and authorisation process works 
- possibly a more extended help function
- possibly the text describing the authorisation on open humans should be revisited to make it shorter and easier to understand

## Who can help
Anyone interested in explaining processes in a short and easy way :)
